src/storyboard/elements.js

```diff
@@ -281,16 +281,298 @@ const StoryboardElements = {
+	}}
+        >
+	    🕮
+        </div>
+        <div style={{
+		fontSize: 16,
+		fontWeight: 'bold',
+		color: Colors.black,
+		float: 'left',
+		position: 'relative',
+		left: 24,
+	    top: -80
+	}}
+	    >
+	    John 3:16
+	</div>
+	    </StoryboardElement>
+    
+    ),
+    'crossLeft' : (
+           <StoryboardElement>
+            <div className='left-vertical-1'
+	       style={{
+		width: 3,
+		height: 40,
+		backgroundColor: Colors.black
+	    }}
+            >
+	    </div>
+	    <div className='left-horizontal-1'
+	        style={{
+		width: 40,
+		height: 3,
+		backgroundColor: Colors.black,
+	        position: 'relative',
+		float: 'left',
+		top: -3,
+		left: -40
+	    }}
+	    >
+	    </div>
+	    <div className='left-vertical-2'
+	        style={{
+		width: 3,
+		height: 40,
+		backgroundColor: Colors.black,
+		position: 'relative',
+		float: 'left',
+		top: 0,
+		left: -80
+	    }}
+	    >
+	    </div>
+	    <div className='left-horizontal-2'
+	        style={{
+		width: 40,
+		height: 3,
+		backgroundColor: Colors.black,
+		position: 'relative',
+	        float: 'left',
+		top: 37,
+		left: -80
+	    }}
+	    >
+	    </div>
+	    <div className='left-vertical-3'
+	        style={{
+		width: 3,
+		height: 96,
+		backgroundColor: Colors.black,
+		position: 'relative',
+		float: 'left',
+		top: 37,
+		left: -83
+	    }}
+	    >
+	    </div>
+	    <div className='cross-down-arrow'
+	style={{
+	    fontSize: 25,
+	    color: Colors.black,
+	    position: 'relative',
+	    float: 'left',
+	    top: 105,
+	    left: -95
+	}}
+	    >
+	    ▼
+	</div>
+	</StoryboardElement>
+    ),
+    'crossRight' : (
+	    <StoryboardElement>
+	    <div className='right-vertical-1'
+	style={{
+	    width: 3,
+	    height: 40,
+	    backgroundColor: Colors.black
+	}}
+	    >
+	    </div>
+	    <div className='right-horizontal-1'
+	style={{
+	    width: 40,
+	    height: 3,
+	    backgroundColor: Colors.black,
+	    position: 'relative',
+	    float: 'left',
+	    top: -3,
+	    left: 0
+	}}
+	    >
+	    </div>
+	    <div className='right-vertical-2'
+	style={{
+	    width: 3,
+	    height: 40,
+	    backgroundColor: Colors.black,
+	    position: 'relative',
+	    float: 'left',
+	    top: -3,
+	    left: 0
+	}}
+	    >
+	    </div>
+	    <div className='right-horizontal-2'
+	style={{
+	    width: 40,
+	    height: 3,
+	    backgroundColor: Colors.black,
+	    position: 'relative',
+	    float: 'left',
+	    top: 37,
+	    left: -40
+	}}
+	    >
+	    </div>
+	    <div className='right-vertical-3'
+	style={{
+	    width: 3,
+	    height: 96,
+	    backgroundColor: Colors.black,
+	    position: 'relative',
+	    float: 'left',
+	    top: 37,
+	    left: -83
+	}}
+	    >
+	    </div>
+	    <div className='cross-up-arrow'
+	style={{
+	    fontSize: 25,
+	    color: Colors.black,
+	    position: 'relative',
+	    float: 'left',
+	    top: -48,
+	    left: -95
+	}}
+	    >
+	    ▲
+	</div>
+	    </StoryboardElement>
+    ),
+    'JESUS' : (
```

Comment:
![image](https://user-images.githubusercontent.com/422501/79943106-ed18d600-841c-11ea-9724-ba997d9cb358.png)

Can we make `JESUS` bigger and adjust the positioning so that it covers up `SIN` better?

`font-size: 30px` and try moving it to `left: 249px; top: 42px;`

Also, for some reason, it doesn't actually look bold, even though the style is applied.